### ISSUE TITLE
Preserve focus in status bar

### ISSUE THREAD

**Issue by leonardder** [microsoft/vscode#97241]
In MS Office, the first entry gets focus by default and if you move focus through items, the focused entry is saved on the status bar so that when you move out/in with f6/shift+f6, the focused item remains the same.

It would be helpful if focus can be retained when the status bar loses focus and gains focus again. This way, you can quickly watch a single entry on the status bar, e.g. horiizontal/vertical position.

As nothing gets focus by default, there should be a way to reset focus as well. May be escape can do this? So if a status bar child is focus, escape can set focus back to the status bar itself.

_Originally posted by @leonardder in https://github.com/microsoft/vscode/issues/95433#issuecomment-625748623_

**Comment by isidorn** [comment]
All this makes sense and I can look into this
fyi @bpasero since you gave some similar feedback

**Comment by isidorn** [comment]
@leonardder thanks for great feedback. I have pushed the following changes:
1) focus on a status bar entry gets preserved when you re-focus the status bar
2) `esc` while in status bar clear the focus from the entry to the whole status bar

I am open to changing that the iital focus auto-focuses the first element, though did not make that change yet since it might break nice voice over behavior  from my previous comment.
